src/admin/admin.js

```diff
@@ -1,33 +1,34 @@
-import { Shop } from '../nimiq-shop.js'
-// console.log(require)
+// TODO(svub) make this proper importing work
+// import Shop from '../nimiq-shop.js'
+// import { Shop2 } from '../nimiq-shop.js'
 
 const { privateKey, publicKey } = localStorage
-const shop = new Shop()
+const shop = new Shop(configuration)
```

Comment:
Where is `configuration` defined?

Reply:
oh, now I understand, yes, admin.js is the old version, updated it a bit here and there but not functional right now. Will do that now as part of the next task. Should have not checked it in. Sorry